bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/TellstickHandlerFactory.java

```diff
@@ -47,8 +48,9 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.tellstick")
 public class TellstickHandlerFactory extends BaseThingHandlerFactory {
```

Comment:
Could this class be annotated with `@NonNullByDefault`?

Reply:
Yes, it can. Done.